sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -122,6 +122,15 @@ public static RequestContent Create(object serializable, JsonPropertyNames prope
             return Create(serializer.Serialize(serializable));
         }
 
+        /// <summary>
+        /// Creates an instance of <see cref="RequestContent"/> that wraps a serialized version of an object.
+        /// </summary>
+        /// <param name="content">The <see cref="PipelineContent"/> to wrap.</param>
+        /// <returns>An instance of <see cref="RequestContent"/> that wraps a serialized version of the object.</returns>
+        [RequiresUnreferencedCode(SerializationRequiresUnreferencedCode)]
+        [RequiresDynamicCode(SerializationRequiresUnreferencedCode)]
+        public static RequestContent Create(PipelineContent content) => new PipelineContentContent(content);
```

Comment:
@annelo-msft I don't think this is how we want to do this longer term but it works for an initial draft to get the alpha out.